platform/migration/platform-migratable-api/src/main/java/org/codice/ddf/migration/Migratable.java

```diff
@@ -18,12 +18,15 @@
 import javax.validation.constraints.NotNull;
 
 /**
- * This interface provides the mechanism for implementers to define how their data shall be
- * exported for later import into a new system.
- *
+ * This interface provides the mechanism for implementers to define how their
+ * components shall be exported for later import into a new system. THIS INTERFACE
```

Comment:
Instead of using ALL CAPS for important information in JavaDoc, we should use standard HTML tags, in this case `<b>text</b>` or `<strong>text</strong>`.